src/index.ts

```diff
@@ -128,11 +129,27 @@ program
       path.join(os.homedir(), 'Desktop', 'cio-sdk-tools-output.logs')
     )
   )
-  .action((path: string, options: DoctorCommandOptions) => {
+  .action(async (path: string, options: DoctorCommandOptions) => {
     configureLogger({
       verbose: options.verbose,
       logFilePath: options.report,
     });
+
+    logger.debug(
+      `Running ${packageJson.name} on version ${packageJson.version}`
+    );
+
+    try {
+      const latestVersion = await fetchCachedLatestVersion(packageJson.name);
+      if (latestVersion && latestVersion !== packageJson.version) {
+        logger.warning(
+          `Newer version of ${packageJson.name} available ${latestVersion}, currently on ${packageJson.version}`
+        );
+      }
+    } catch {
+      // catch block is empty because fetchLatestVersion will log the error
+    }
+
```

Comment:
I think we don't need to wrap this in try catch because `fetchCachedLatestVersion` is already returning `undefined` for handling failures gracefully.